docs/security.md

```diff
@@ -75,6 +76,19 @@ PRIVATE_KEY=key flow project deploy
 }
 ```
 
+### Private Environment File
+
+Setting environment variables can be avoided by using environment file `.env`. Anything 
+you put in `.env` file will be loaded as environment variable and replaced in configuration the 
+same way as environment variable.
+
+⚠️ Please be sure to add `.env` file to `.gitignore`
```

Comment:
```suggestion
⚠️ You should never commit `.env` to source control, especially if it contains sensitive information like a private key.
```

Just because not all users use git